Waterfall.h

```diff
@@ -272,7 +272,7 @@ class Waterfall : public QFrame
     }
 
     void setFftCenterFreq(qint64 f) {
-        qint64 limit = ((qint64)m_SampleFreq + m_Span) / 2 - 1;
+        qint64 limit = ((qint64)m_SampleFreq - m_Span) / 2 - 1;
```

Comment:
What happens if the sample rate equals the span?

Reply:
Ah, good catch, an off-by-one causes the qBound minimum to be greater than the maximum when fully zoomed out. I didn't catch it because I only did a release build that skips that assertion.

Reply:
I think just removing the minus one should fix it, though I'll test and ponder more tonight about the original purpose of the minus one.